src/utils/db/car_index/car_index_builder.rs

```diff
@@ -141,4 +141,8 @@ impl CarIndexBuilder {
     pub fn len(&self) -> u64 {
         self.table.len() as u64
     }
+
+    pub fn is_empty(&self) -> bool {
+        self.table.is_empty()
+    }
```

Comment:
Where is this function used?

Reply:
This is to fix a clippy warning after `mod util` has been made public to be accessible in tests/*

```
warning: struct `CarIndexBuilder` has a public `len` method, but no `is_empty` method
   --> src/utils/db/car_index/car_index_builder.rs:141:5
    |
141 |     pub fn len(&self) -> u64 {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#len_without_is_empty
    = note: `#[warn(clippy::len_without_is_empty)]` on by default
```

Reply:
Let's add an exemption. This definition of `is_empty` is not meaningful.

Reply:
Can also rename `len` to `capacity`.

Reply:
Renamed to `capacity`